src/easypost.js

```diff
@@ -173,12 +158,6 @@ export default class API {
       ...additionalHeaders,
     };
 
-    if (API.runningInBrowser()) {
```

Comment:
# Reviewer Note

I accidentally introduced this function but had it flipped (it would remove headers for Node requests and keep them for browser which was backwards); however, this didn't get released yet so it was caught in time.